base58/__init__.py

```diff
@@ -65,21 +65,30 @@ def b58encode(
 
 
 @lru_cache()
-def _get_base58_decode_map(alphabet: bytes) -> Mapping[int, int]:
-    map = {char: index for index, char in enumerate(alphabet)}
-    return map
+def _get_base58_decode_map(alphabet: bytes, autofix: bool) -> Mapping[int, int]:
+    invmap = {char: index for index, char in enumerate(alphabet)}
+
+    if autofix:
+        groups = [b'0Oo', b'Il1']
+        for group in groups:
+            pivots = list(filter(lambda x: x in alphabet, group))
```

Comment:
A list comprehension is easier to use here and a lookup in the map faster
```suggestion
            pivots = [c for c in alphabet if c in invmap]
```